lemur/manage.py

```diff
@@ -126,7 +126,9 @@ def cli(script_info, config):
 LOG_LEVEL = "DEBUG"
 LOG_FILE = "lemur.log"
 LOG_UPGRADE_FILE = "db_upgrade.log"
-
+LOG_REQUEST_HEADERS = False
+LOG_SANITIZE_REQUEST_HEADERS = True
+LOG_REQUEST_HEADERS_SKIP_ENDPOINT = ["/metrics", "/healthcheck"]  # These endpoints are noisy so skip them by default
```

Comment:
Can you please document these configs under Lemur Docs https://lemur.readthedocs.io/en/latest/administration.html

Reply:
@charhate great call out, thanks for catching that. I've updated the docs, let me know if any need more clarification or additional verbage.